<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confirmationDialog</class>
 <widget class="QDialog" name="confirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>180</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPlainTextEdit" name="messageLogTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18pt;
border: 1px dash rgb(132, 198, 198);
background-color: white;

</string>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::WidgetWidth</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="cursorWidth">
    <number>0</number>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleMessage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 82, 143);
color: rgb(255, 255, 255);
font-size:22pt;
padding: 0 0 0 10;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="logIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>info_icon2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="recogLogWindow_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px dash rgb(255, 255, 255);
background-color: rgb(239, 239, 239);

</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <zorder>recogLogWindow_2</zorder>
  <zorder>messageLogTextEdit</zorder>
  <zorder>titleMessage</zorder>
  <zorder>logIcon</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
